extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -261,7 +261,7 @@ export class ProjectsController {
 		}
 
 		// get dlls and targets file needed for building for legacy style projects
-		if (!project.isSdkStyleProject) {
+		if (project.sqlProjStyle === mssql.ProjectType.LegacyStyle) {
```

Comment:
reminder to go back and handle vscode-mssql for all these types that defined in both mssql.d.ts and vscode-mssql.d.ts

Reply:
How does that typically work for something like this?

Reply:
I think we'll have to do something like this for all the calls to STS. Maybe we should just do that all in one pass at the end after we're sure everything works in ADS first
https://github.com/microsoft/azuredatastudio/blob/3481a6b97386cba9144c56d933c7d098daecfbd7/extensions/sql-database-projects/src/controllers/projectController.ts#L1564-L1568

Reply:
Yeah, probably easier/cleaner to do that on settled code.  I made a backlog item for that in March.